robosats/urls.py

```diff
@@ -15,19 +15,30 @@
 """
 
 from decouple import config
+from django.apps import apps
 from django.conf import settings
-from django.contrib import admin
 from django.urls import include, path
 
+from django_otp.admin import OTPAdminSite
+
+
+class OTPAdmin(OTPAdminSite):
+    pass
+
+
+admin_site = OTPAdmin(name="OTPAdmin")
+for model in apps.get_models():
+    admin_site.register(model)
```

Comment:
This includes more (hidden) models than the normal admin view, but honestly I found them useful